.github/workflows/release.yml

```diff
@@ -0,0 +1,129 @@
+name: CLI Release
+
+on:
+  workflow_dispatch:
+  # schedule:
+  #   - cron: '0 12 * * 2'  # Runs every Tuesday at 12:00 PM (noon) ET
+
+jobs:
+
+  checks:
+    runs-on: ubuntu-latest
+    outputs:
+      HAS_TAG: ${{ steps.check_tag.outputs.has_tag }}
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+      
+      - name: Check if commit has a new tag
```

Comment:
You can change this to be a trigger on a tag. Instead of having it be a check against tags